ApolloExtra.lua

```diff
@@ -3809,6 +3842,35 @@ function CheckHyacinthKill( args, attacker, victim )
 	end
 end
 
+-- Archery Lessons damage resist function mod
+ModUtil.WrapBaseFunction( "OnProjectileReflect", 
```

Comment:
This shouldn't work since OnProjectileReflect is not a function, do a test to see if this gets called on Reflect. What you need to do is OnProjectileReflect {function(triggerArgs) [[DO STUFF]]   end}
This will run in parallel to the other OnProjectileReflect.